fenix/app/src/androidTest/java/org/mozilla/fenix/ui/ComposeTabbedBrowsingTest.kt

```diff
@@ -375,4 +375,17 @@ class ComposeTabbedBrowsingTest {
             verifyNavigationToolbar()
         }
     }
+
+    @Test
+    fun verifySyncedTabsWhenUserIsNotSignedInTest() {
+        navigationToolbar {
+        }.openComposeTabDrawer(composeTestRule) {
+            verifySyncedTabsButtonIsSelected(false)
```

Comment:
Reminder about the named arguments for better readability

Reply:
Done !